src/components/MovieCard/MovieCard.scss

```diff
@@ -1 +1,9 @@
-// 
+.card-image img {
```

Comment:
```suggestion
.card__image img {
```

set `object-fit` property to card image, pay attention to your class name 

and then you can remove redundant styles for `.image`

Reply:
I understand, thank you